aviary/mission/two_dof_problem_configurator.py

```diff
@@ -350,7 +352,7 @@ def set_phase_options(self, prob, phase_name, phase_idx, phase, user_options):
             )
 
         elif 'descent' in phase_name:
-            duration_ref = user_options.get_val('duration_ref', 's')
+            duration_ref = user_options['time_duration_ref'][0]
```

Comment:
are we going to be ok with unit conversions here? This is hard-coded to seconds but what if the user inputs minutes?